ibazel/command/notify_command.go

```diff
@@ -89,6 +89,12 @@ func (c *notifyCommand) Start() (*bytes.Buffer, error) {
 }
 
 func (c *notifyCommand) NotifyOfChanges() *bytes.Buffer {
+	if !c.IsSubprocessRunning() {
```

Comment:
I think we should also print out a log message informing the user that we are restarting their subprocess

Reply:
Done